etc/di.xml

```diff
@@ -24,4 +24,15 @@
     <type name="Magento\Framework\App\FrontControllerInterface">
         <plugin name="sansec_shield" type="Sansec\Shield\Plugin\ShieldPlugin"/>
     </type>
+    <type name="Sansec\Shield\Model\IP">
```

Comment:
why did you move this from code to di?

Reply:
Well I thought those headers should be injected via the constructor, so that it can be modified/extended by consumers. di.xml is just how magento does it

Reply:
For example anyone else can now simply do:

```xml
<type name="Sansec\Shield\Model\IP">
        <arguments>
            <argument name="ipHeaders" xsi:type="array">
                <item name="MY_CUSTOM_HEADER" xsi:type="string">MY_CUSTOM_HEADER</item>
            </argument>
        </arguments>
</type>
```

Without having to create a plugin/override.